### ISSUE TITLE
Image management lacking

### ISSUE THREAD

**Issue by KamillaPup** [pornganizer/PGV5#56]
I was playing around with this a bit and like all the features but not a fan of the UI or implementation of it.

1. I would like to tag multiple things at once, especially images, and making image sets doesn't seem to be possible. Most of my collection are images so dragging and dropping a named folder, it should be able to keep those images in a set/album.
2. I can't see a way to view the tags in the details pane that are applied.
3. Is there no feature to rename/organize files on the hard drive based on the tagging in the database? For instance /website/model/set or something else similar.

Thanks! I'm looking forward to how well this can work. Right now, I'm using Media Center 27 but it's really cumbersome and I would love a program built for smut.

**Comment by WeedLordVegeta420** [comment]
> 1. I would like to tag multiple things at once, especially images, and making image sets doesn't seem to be possible.

I noticed that this is actually already possible but just needs to be improved. You can CTRL+Click and Shift+Click to select multiple items or CTRL+A to select all and then apply a tag. The biggest issue is that there needs to be some sort of highlight on the selected item so you can actually see it.

I do second the need for a better way to create image sets. I also would like to have the ability to set a default image for a Scene or directly associate an Image with a Scene. Clearly some of those associations exist under the hood since capturing a frame ties the image to a scene in some way, but we can't manually edit them or even see them.

For context, for JAV stuff that's all distributed as a full DVD rip, I'm trying to avoid creating DVDs that all have a single Scene just to have a way to display the DVD cover scan. I would prefer to just directly set the DVD cover as the thumbnail for the Scene.

On the topic of capturing a frame, it would also be nice if I could jump directly to the portion of the video where that frame was captured.

**Comment by pornganizer** [comment]
> I noticed that this is actually already possible but just needs to be improved. You can CTRL+Click and Shift+Click to select multiple items or CTRL+A to select all and then apply a tag. The biggest issue is that there needs to be some sort of highlight on the selected item so you can actually see it.

correct. i'll have to revisit the selection indicator. the problem with a standard border is that it consumes space when not there and just make the display awkward most of the time.

> I do second the need for a better way to create image sets. I also would like to have the ability to set a default image for a Scene or directly associate an Image with a Scene. Clearly some of those associations exist under the hood since capturing a frame ties the image to a scene in some way, but we can't manually edit them or even see them.

ill work on scene image sets for the next release

> For context, for JAV stuff that's all distributed as a full DVD rip, I'm trying to avoid creating DVDs that all have a single Scene just to have a way to display the DVD cover scan. I would prefer to just directly set the DVD cover as the thumbnail for the Scene.

this is awkward because a scene tile is always landscape, but a dvd is portrait

> On the topic of capturing a frame, it would also be nice if I could jump directly to the portion of the video where that frame was captured.

category bookmarks allow you to pinpoint moments in a scene and jump between them. i'd think this is better then being able to jump to where you took an image from?

**Comment by WeedLordVegeta420** [comment]
Fair point on the DVD cover aspect ratio and bookmarks.
